bindings/python/manifold3d.cpp

```diff
@@ -315,6 +315,14 @@ NB_MODULE(manifold3d, m) {
       .def("calculate_curvature", &Manifold::CalculateCurvature,
            nb::arg("gaussian_idx"), nb::arg("mean_idx"),
            manifold__calculate_curvature__gaussian_idx__mean_idx)
+      .def(
+          "min_gap",
```

Comment:
Can you add this to `all_apis.py` as well, for at least a trivial test?

Reply:
Done!